src/Turn.js

```diff
@@ -1,29 +1,27 @@
 import domUpdates from './domUpdates';
 
 class Turn {
-  constructor(game, round) {
-    this.game = game;
-    this.round = round;
-    this.currentPlayer = this.game.playerOne;
+  constructor() {
   }
 
-  checkGuess(guess) {
-    let index = this.round.currentSurveyAnswers.findIndex(answerObj => answerObj.answer.toLowerCase() === guess.toLowerCase());
+  checkGuess(game, guess) {
+    let index = game.currentRound.currentSurveyAnswers.findIndex(answerObj => answerObj.answer.toLowerCase() === guess.toLowerCase());
     if (index !== -1) {
-      let targetAnswer = this.round.currentSurveyAnswers.splice(index, 1)[0];
-      this.assignPoints(targetAnswer);
+      let targetAnswer = game.currentRound.currentSurveyAnswers.splice(index, 1)[0];
+      this.assignPoints(game, targetAnswer);
     }
-    this.endTurn();
+    this.endTurn(game);
   }
 
-  assignPoints(targetAnswer) {
-    this.currentPlayer.score += targetAnswer.respondents * this.currentPlayer.multiplier;
+  assignPoints(game, targetAnswer) {
+    game.currentRound.currentPlayer.score += targetAnswer.respondents * game.currentRound.currentPlayer.multiplier;
   }
 
-  endTurn() {
-    this.currentPlayer === this.game.playerOne ? this.currentPlayer = this.game.playerTwo : this.currentPlayer = this.game.playerOne;
-    if (this.round.currentSurveyAnswers.length === 0) {
-      this.round.endRound();
+  endTurn(game) {
+    game.currentRound.currentPlayer === game.playerOne ? game.currentRound.currentPlayer = game.playerTwo : game.currentRound.currentPlayer = game.playerOne;
+    if (game.currentRound.currentSurveyAnswers.length === 0) {
```

Comment:
0 is falsey in JavaScript so you could just say `if(game.currentRound.currentSurveyAnswers.length)`